src/components/dash_data/ComparisonGrid.tsx

```
@@ -0,0 +1,248 @@
+
+export interface CompanyDataEx {
+  name: string;
+  ticker: string;
+  revenue: number[];
+  profit: number[];
+  assets: number[];
+  liabilities: number[];
+  years: string[];
+}
+
+export const MOCK_COMPANIES: CompanyDataEx[] = [
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)